crates/uv-platform-tags/src/abi_tag.rs

```diff
@@ -0,0 +1,445 @@
+use std::fmt::Formatter;
+use std::str::FromStr;
+
+/// A tag to represent the ABI compatibility of a Python distribution.
+///
+/// This is the second segment in the wheel filename, following the language tag. For example,
+/// in `cp39-none-manylinux_2_24_x86_64.whl`, the ABI tag is `none`.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum AbiTag {
+    /// Ex) `none`
+    None,
+    /// Ex) `abi3`
+    Abi3,
+    /// Ex) `cp39m`, `cp310t`
+    CPython {
+        gil_disabled: bool,
```

Comment:
Just wondering, why this naming? We use "freethreaded" elsewhere and that avoids a double-negative (e.g., gil disabled = false is slightly harder to reason about)

Reply:
Dear goodness it comes from `Py_GIL_DISABLED` and is present elsewhere in our code. I defer to you on what's best here, but have a preference for "freethreaded"

Reply:
Lol

Reply:
I'll just leave it for now and revisit in a separate PR.